website/src/content/docs/components/card.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: Card
+description: Cards contain content and actions about a single subject.
+---
+
+Cards are surfaces that display content and actions on a single topic. They should be easy to scan for relevant and actionable information. Elements, like text and images, should be placed on them in a way that clearly indicates hierarchy.
+
+## Usage
+
+```astro live
+---
+import { Card } from "@gemini-ui-astro/components";
+import { Button } from "@gemini-ui-astro/components";
+---
+
+<Card class="example-card" dynamicGlow style={{ marginBottom: "32px" }}>
```

Comment:
We can fix this later but the card text has low contrast with the background in light mode.

Reply:
Yeah we need to implement a light / dark mode